horizons/ai/aiplayer/productionbuilder.py

```diff
@@ -263,24 +263,24 @@ def display(self):
 			return
 
 		tile_colors = {
-			BUILDING_PURPOSE.ROAD:            ( 30,  30,  30),
```

Comment:
Hm. So leading zeroes would be okay, but leading spaces aren't? The hex values are more readable or helpful than the numeric ones?

There's also this little problem:

```py
>>> 030 == 30
False
```